spanner/stats.go

```diff
@@ -213,28 +216,40 @@ func EnableStatViews() error {
 
 // EnableGfeLatencyView enables GFELatency metric
 func EnableGfeLatencyView() error {
-	GFELatencyMetricsEnabled = true
+	SetGFELatencyMetricsFlag(true)
 	return view.Register(GFELatencyView)
 }
 
 // EnableGfeHeaderMissingCountView enables GFEHeaderMissingCount metric
 func EnableGfeHeaderMissingCountView() error {
-	GFELatencyMetricsEnabled = true
+	SetGFELatencyMetricsFlag(true)
 	return view.Register(GFEHeaderMissingCountView)
 }
 
 // EnableGfeLatencyAndHeaderMissingCountViews enables GFEHeaderMissingCount and GFELatency metric
 func EnableGfeLatencyAndHeaderMissingCountViews() error {
-	GFELatencyMetricsEnabled = true
+	SetGFELatencyMetricsFlag(true)
 	return view.Register(
 		GFELatencyView,
 		GFEHeaderMissingCountView,
 	)
 }
 
+func GetGFELatencyMetricsFlag() bool {
```

Comment:
Could this be unexported? And same for the setter?

Reply:
done